LICENSE-binary

```diff
@@ -216,29 +216,29 @@ commons-lang3-3.12.0
 commons-logging-1.2
 commons-validator-1.7
 error_prone_annotations-2.10.0
-jackson-annotations-2.16.1
-jackson-core-2.16.1
-jackson-databind-2.16.1
-jackson-dataformat-csv-2.16.1
-jackson-datatype-jdk8-2.16.1
-jackson-jaxrs-base-2.16.1
-jackson-jaxrs-json-provider-2.16.1
-jackson-module-afterburner-2.16.1
-jackson-module-jaxb-annotations-2.16.1
-jackson-module-scala_2.13-2.16.1
-jackson-module-scala_2.12-2.16.1
+jackson-annotations-2.16.2
+jackson-core-2.16.2
+jackson-databind-2.16.2
+jackson-dataformat-csv-2.16.2
+jackson-datatype-jdk8-2.16.2
+jackson-jaxrs-base-2.16.2
+jackson-jaxrs-json-provider-2.16.2
+jackson-module-afterburner-2.16.2
+jackson-module-jaxb-annotations-2.16.2
+jackson-module-scala_2.13-2.16.2
+jackson-module-scala_2.12-2.16.2
 jakarta.validation-api-2.0.2
```

Comment:
Are these intended changes, if yes then we shall have it in separate PR.

Reply:
Yes, the commit history got changed a bit. It should be fine now with the latest push in place